packages/oas/test/operation/lib/get-response-as-json-schema.test.ts

```diff
@@ -457,4 +458,102 @@ describe('#getResponseAsJSONSchema()', () => {
       });
     });
   });
+
+  describe('discriminator + allOf inheritance', () => {
+    async function loadDiscriminatorSpec() {
+      const data = await import('../../__datasets__/discriminator-allof-inheritance.json').then(r => r.default);
+      // Deep clone to avoid mutation between tests
+      return Oas.init(JSON.parse(JSON.stringify(data)));
+    }
```

Comment:
I'm assuming the `useImportExtensions` ignore at the top of the file is because of the JSON import in here? Instead of that, can we do a common import of this JSON file at the top and then replace this method with the following?

```suggestion
    function loadDiscriminatorSpec() {
      return Oas.init(structuredClone(discrimnatorAllOfInheritance));
    }
```